sktime/forecasting/compose/tests/test_multiplex.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3 -u
+# -*- coding: utf-8 -*-
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Tests for MultiplexForecaster and associated dunders."""
+
+__author__ = ["aiwalter", "miraep8"]
```

Comment:
why is aiwalter an author? Did you take code from elsewhere?

Reply:
Sorry - chalk this up to me being new to collaborative coding/open source projects!  I wanted to ere on the side of giving as much credit as possible (as I mentioned in the **Any other comments** section above!) But no - the code including the unfortunate reference error is mine! (though I certainly did look at the example `MultiplexForecaster` code as well as other test scripts for inspiration!).

Reply:
so, unless @aiwalter objects, I´d just suggest to remove it.

This is not like academia where you have to put the department head, their cousin and their dog on the paper 😄 

Also, whoever is in the author´s list is the person to blame for bugs in the file 😁

Reply:
It has been removed!   Blame me for the bugs!! 🐛 🐛 🐛